ui/src/app/components/help-sidebar.tsx

```diff
@@ -524,15 +540,69 @@ export const HelpSidebar = fp.flow(withCurrentWorkspace(), withUserProfile(), wi
       return !icon.page || icon.page === helpContentKey || (criteria && icon.page === 'criteria');
     }
 
-    displayIcon(icon, i) {
-      const {criteria} = this.props;
+    displayFontAwesomeIcon(icon) {
+      const {criteria, runtimeStore} = this.props;
 
       return <React.Fragment>
-        {(criteria && icon.page === 'criteria') && <span data-test-id='criteria-count'
-                                                         style={styles.criteriaCount}>
-          {criteria.length}</span>}
-            <FontAwesomeIcon data-test-id={'help-sidebar-icon-' + icon.id} icon={icon.faIcon} style={icon.style} />
-          </React.Fragment> ;
+        {(criteria && icon.page === 'criteria') &&
+          <span data-test-id='criteria-count' style={styles.criteriaCount}>
+            {criteria.length}
+          </span>
+        }
+        <FontAwesomeIcon data-test-id={'help-sidebar-icon-' + icon.id} icon={icon.faIcon} style={icon.style} />
+      </React.Fragment> ;
+    }
+
+    displayRuntimeIcon(icon) {
+      const {runtimeStore} = this.props;
+
+      return <FlexRow style={{height: '100%', alignItems: 'center', justifyContent: 'space-around'}}>
+        <img data-test-id={'help-sidebar-icon-' + icon.id} src={proIcons[icon.id]} style={{...icon.style, position: 'absolute'}} />
+        {
```

Comment:
Can you put a comment here to explain the scope of this big conditional, and specifically what should happen if we see a status which is not covered here? My understanding is that the icon above is always shown, and then there may optionally be a status indicator shown as an overlay.